### ISSUE TITLE
Notched T wave

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7593]
**Preferred term label:**
Notched T wave

**Synonyms**
Notched T-wave

**Definition (free text, please give PubMed ID)**
A bifid T-wave with a notch duration between the 2 peaks at leasy 0.04 sec and an amplitude at least 0.05 mV. 
PMID: 29699857
**Parent term (use hpo.jax.org/app)**
Abnormal T-wave HP:0005135

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 613688 
PMID:34357002 3/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
